### PR TITLE
Yo

### PR DESCRIPTION
**Pull request by CasperBrun**

### CODE REVIEW

File: tests/test_make_dataset.py [hunk 1]
```diff
@@ -12,6 +12,14 @@ def test_get_tokenizer():
     # delete "albert-base-v2" and run again
 
 
+def test_merge_csv():
+    assert type(merge_csv()) == pd.DataFrame
```
Comment (RSM-git) [inline tests/test_make_dataset.py]: change type equality check to isinstance(merge_csv(), pd.DataFrame)

**Review by RSM-git — CHANGES_REQUESTED** [review]

File: src/data/make_dataset.py [hunk 2]
```diff
@@ -79,6 +80,37 @@ def get_tokenizer() -> transformers.AlbertTokenizerFast:
     return tokenizer
 
 
+def merge_csv() -> pd.DataFrame:
+
```
Comment (Tolsager) [inline src/data/make_dataset.py:86]: delete empty line

File: tests/test_make_dataset.py [hunk 3]
```diff
@@ -12,6 +12,14 @@ def test_get_tokenizer():
...
+
+
+def test_split():
+    pass
```
Comment (Tolsager) [inline tests/test_make_dataset.py]:
```suggestion
    assert isinstance(split(), tuple)
```

**Review by Tolsager — CHANGES_REQUESTED** [review]

File: tests/test_make_dataset.py (hunk 1 above)
Comment (RSM-git) [inline tests/test_make_dataset.py]:
```suggestion
    assert isinstance(merge_csv(), pd.DataFrame)
```

**Review by RSM-git — CHANGES_REQUESTED** [review]

**Pull request closed without merging**